docs/static/ls-ls-native.asciidoc

```diff
@@ -36,15 +36,13 @@ input {
 
 In order to obtain the best performance when sending data from one Logstash to another, the data is batched and compressed. As such, the upstream Logstash (the sending Logstash) only needs to be concerned about configuring the receiving endpoint with these options:
 
-* `hosts` - The receiving Logstash and port. If no port specified, 9800 will be used.
-
-NOTE: In the future, {ls} will support multiple output hosts.
```

Comment:
Maybe we should change the note to say something like:

NOTE: Logstash supports multiple host by automatically loadbalancing events across all destination ports.

Or similar, but not a requirement.